### PR TITLE
feature/sd-84: Search results.

### PR DESCRIPTION
**Pull request by MasseyIsaako**

### CODE REVIEW

File: templates/Includes/SearchResult.ss [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<article class="listing__item" data-highlight="$Up.Query.ATT">
```
Comment (adunn49) [inline templates/Includes/SearchResult.ss]: just a reminder to remove the $Up from this line

File: templates/Layout/Page_results.ss [hunk 2]
```diff
@@ -1,16 +1,17 @@
 <div class="content search-results">
     <div class="container">
         <div class="row">
-            <section class="col-lg-10 offset-lg-1">
-                <div class="pb-2 mt-4 mb-4 pb-3 border-bottom">
+            <section class="col-lg-8">
+                <%-- <div class="pb-2 mt-4 mb-4 pb-3 border-bottom"> --%>
```
Comment (adunn49) [inline templates/Layout/Page_results.ss]: can remove this

File: templates/Layout/Page_results.ss [hunk 3]
```diff
@@ -30,7 +31,7 @@
                                 </div>
                                 <div class="col-sm-12 col-md-4 float-right search-results-results-page">
                                     <p class="text-muted">
-                                        <%t CWP_Search.Pages "Displaying Page {current} of {total}" current=$Results.CurrentPage total=$Results.TotalPages %>
+                                        <%t CWP_Search.Pages "Displaying {current} of {total}" current=$Results.CurrentPage total=$Results.TotalPages %>
```
Comment (adunn49) [inline templates/Layout/Page_results.ss]: With Page removed its ambiguous whether it means results or pages. (so maybe split this out into separate template for watea, not sure)

File: templates/SilverStripe/CMS/Search/Includes/SearchForm.ss [hunk 4]
```diff
@@ -9,7 +9,7 @@
             placeholder="<%t CWP_Search.Placeholder "Search..." %>"/>
         <label class="sr-only" for="SearchForm_SearchForm_Search"><%t CWP_Search.Button "Search" %></label>
         <div class="input-group-append">
-            <button type="submit" class="btn btn-secondary btn-lg"><%t CWP_Search.Button "Search" %></button>
+            <button type="submit" class="btn btn-secondary btn-lg search__btn"><%t CWP_Search.Button "Search" %></button>
```
Comment (adunn49) [inline templates/SilverStripe/CMS/Search/Includes/SearchForm.ss]: Interestingly, if you remove the search__btn class and replace btn-secondary with btn-primary you get the styling you were after. I wonder if this should be changed to a primary button.

File: templates/Includes/SearchResult.ss (hunk 1 above)
Comment (MasseyIsaako) [reply templates/Includes/SearchResult.ss]: This template has been removed

File: templates/Layout/Page_results.ss (hunk 2 above)
Comment (MasseyIsaako) [reply templates/Layout/Page_results.ss]: Done!

File: templates/Layout/Page_results.ss (hunk 3 above)
Comment (MasseyIsaako) [reply templates/Layout/Page_results.ss]: Done!

**Pull request merged**